doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -156,23 +156,69 @@ The below set of steps define the compliance, verification and certification pro
 
 <a name="2.7"></a>
 ## 2.7 Frameworks
-The NFVI certification framework deals with the process of testing NFVI in below three areas:
-* Compliance: The NFVI needs to comply to CNTT RA1/RA2.
-* Validation: Validation deals with the ability of NFVI to respond to Cloud APIs and interfaces.
-* Performance: Performance deals with running tests on NFVI depending on the NFVI profile and collecting KPIs.
 
-The NFVI KPIs are compared with Golden KPIs, which serve as a reference for NFVI certification. If NFVI KPIs meet Golden KPIs, NFVI is certified and granted a certification badge. If NFVI KPIs do not meet Golden KPIs, no certification badge is provided.
```

Comment:
Keep, as this is general / generic guidance and principles for Compliance "Frameworks".

Reply:
Agree It should be moved out of Framework (guidelines).
Updating.

Reply:
revert even if I disagree here about the framework structure.

Reply:
revert even if I disagree here about the framework structure.